### ISSUE TITLE
Not enough LSK to send transaction

### ISSUE THREAD

**Issue by ghost** [fix/Lisk-Lite-Client#11]
Can't send transaction if transaction amount is equal to current balance but if i deduce requested amount by 0.1 sending is possible but wallet deduce it again.

**Comment by fix** [comment]
yes current the fees are taken from the sender, not the receiver.

This is more UX than core. I will think about a better approach
